src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Compiler/AddConsoleCommandPass.php

```diff
@@ -37,7 +37,11 @@ public function process(ContainerBuilder $container)
             if (!is_subclass_of($class, 'Symfony\\Component\\Console\\Command\\Command')) {
                 throw new \InvalidArgumentException(sprintf('The service "%s" tagged "console.command" must be a subclass of "Symfony\\Component\\Console\\Command\\Command".', $id));
             }
-            $container->setAlias($serviceId = 'console.command.'.strtolower(str_replace('\\', '_', $class)), $id);
+            $serviceId = 'console.command.'.strtolower(str_replace('\\', '_', $class));
+            if ($container->hasAlias($serviceId)) {
```

Comment:
It uses the service name in the alias. For working services, the `$id` has to be unique, which is given in the alias too.